Integrations/MimecastV2/MimecastV2.py

```diff
@@ -1479,6 +1479,101 @@ def download_attachment_request(attachment_id):
     return response.content
 
 
+def search_mail():
+    """ Newer endpoint to search for mails """
+    headers = []  # type: List[Any]
+    contents = []  # type: List[Any]
+    context = {}  # type: Dict[Any, Any]
+    search_params = {}  # type: Dict[Any, Any]
+    advanced_params = {}  # type: Dict[Any, Any]
+    search_reason = demisto.args().get('search_reason', '').encode('utf-8')
+    start = demisto.args().get('start', '').encode('utf-8')
+    end = demisto.args().get('end', '').encode('utf-8')
+    sender_ip = demisto.args().get('sender_ip', '').encode('utf-8')
+    mail_to = demisto.args().get('mail_to', '').encode('utf-8')
+    mail_from_ = demisto.args().get('mail_from', '').encode('utf-8')
+    subject = demisto.args().get('subject', '').encode('utf-8')
+    message_id = demisto.args().get('message_id', '').encode('utf-8')
+    limit = int(demisto.args().get('limit', 100))
+
+    if search_reason:
+        search_params['searchReason'] = search_reason
+    if start:
+        search_params['start'] = start
+    if end:
+        search_params['end'] = end
+    if message_id:
+        search_params['messageId'] = message_id
+    if mail_from_:
+        advanced_params['from'] = mail_from_
+    if mail_to:
+        advanced_params['to'] = mail_to
+    if subject:
+        advanced_params['subject'] = subject
+    if sender_ip:
+        advanced_params['senderIP'] = sender_ip
+
+    search_params['advancedTrackAndTraceOptions'] = advanced_params
+    tracked_mails = search_mail_request(search_params)
+    if limit:
+        tracked_mails = tracked_mails[:limit]
```

Comment:
lets verify `tracked_mails` is list

Reply:
Now the tracked emails will be list also if nothing returns, so no need to revert